src/client/Microsoft.Identity.Client/TokenSource.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Specifies the source of the access and Id tokens in the authentication result.
```

Comment:
I made some changes to the comments - please have a look. I though GitHub would "propose changes"... I guess I'm not a GitHub power user...

Reply:
lol. 

---
In reply to: [453897525](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1933#discussion_r453897525) [](ancestors = 453897525)

Reply:
You need to use the +/- icon to proposed suggestions, @bgavrilMS 
![image](https://user-images.githubusercontent.com/13203188/87402130-69096100-c5bb-11ea-8608-ce1a1ee34160.png)